src/CONST.ts

```diff
@@ -647,6 +647,7 @@ const CONST = {
         MAX_PREVIEW_AVATARS: 4,
         MAX_ROOM_NAME_LENGTH: 79,
         LAST_MESSAGE_TEXT_MAX_LENGTH: 200,
+        MAX_AMOUNT_OF_RECENTLY_ACCESSED_REPORT_IDS_IN_STACK: 10,
```

Comment:
```suggestion
        MAX_RECENTLY_ACCESSED_REPORT_IDS_STACK_LIMIT: 10,
```

can we name this something like this?

Reply:
Great! Updated.